packages/admin/app/lib/apolloClient.ts

```diff
@@ -0,0 +1,6 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+
+export const apolloClient = new ApolloClient({
+  cache: new InMemoryCache({}),
```

Comment:
You're almost certainly going to use this in loaders/actions exclusively. So you definitely don't want any cache. This is because there's the chance that multiple users could access the same instance this is running in, and you could get nasty side effects because of that.

Reply:
@cairin I see that a cache is required in the constructor for Apollo Clients but you can set the fetch policies to `no-cache`. Is that sufficient?
```
new ApolloClient({
    cache: new InMemoryCache({}),
    defaultOptions: {
      query: {
        fetchPolicy: 'no-cache'
      },
      mutate: {
        fetchPolicy: 'no-cache'
      }
    },
    uri: 'http://localhost:3001/graphql'
  })
```